venafi/resource_venafi_certificate.go

```diff
@@ -60,6 +60,12 @@ func resourceVenafiCertificate() *schema.Resource {
 				Description: "Common name of certificate",
 				ForceNew:    true,
 			},
+			"object_name": &schema.Schema{
```

Comment:
I think we should call this `nickname` to be consistent with [TPP (Aperture)](https://docs.venafi.com/Docs/current/TopNav/Content/Aperture/t-get-new-cert-aperture-tpp.php?Highlight=nickname) and [VCert CLI](https://github.com/Venafi/vcert/blob/master/cmd/vcert/flags.go#L130).

Reply:
I was setting `object_name` since it seemed what most users of our provider will look into when they navigate through TPP documentation. Is there any reason why it was renamed from `object_name` to `nickname` in VCert CLI?

Reply:
Yes, it's because TPP moved away from a object/tree based user experience several years ago.  There are, of course, still certificate objects in the TPP policy tree but that detail is considered more of a "behind the scenes" or advanced usage thing.

Reply:
Got it, will do the renaming in the next commit.